docs/framework/install/on-windows-10.md

```diff
@@ -53,12 +54,24 @@ Follow the instructions to install the [.NET Framework 3.5 on Windows 10](dotnet
 
 The .NET Framework 3.5 supports apps built for .NET Framework 1.0 through 3.5.
 
+## Additional information
+
+.NET Framework 4.x versions are in-place updates to earlier versions. That means the following:
+
+- You can only have one version of .NET Framework 4.x installed on your machine.
+
+- You cannot install an earlier version of the .NET Framework in your machine if a later version is already installed.
+
+- 4.x versions of the .NET Framework can be used to run applications built for .NET Framework 4.0 through that version. For example, .NET Framework 4.7 can be used to run applications built for .NET Framework 4.0 through 4.7.
```

Comment:
Perhaps add "The latest version (the .NET Framework 4.7.1) can be used to run applications built will all versions of the .NET Framework starting with 4.0.